gha-webhook-listener.py

```diff
@@ -251,8 +253,14 @@ def deploy_tarball(artifact_url: str, target_dir: str) -> None:
                 raise
 
     logger.info("...download complete.")
+    if arg_symlink:
+        create_symlink(source=target_dir, linkname=arg_symlink)
 
-    create_symlink(source=target_dir, linkname=arg_symlink)
+    if arg_hook_script is not None:
+        logger.info("...running hook script")
+        return_code = subprocess.run([*arg_hook_script.split(), target_dir]).returncode
```

Comment:
I'd do this to avoid the `*` magic:
```suggestion
        return_code = subprocess.run(arg_hook_script.split() + [target_dir]).returncode
```

Reply:
also, you probably want [`shlex.split`](https://docs.python.org/3/library/shlex.html#shlex.split) rather than regular split here, just so people have a way to pass arguments/script names with spaces in, if they really want to.